whereby/sdk
# Next.js example app

### Description
Provides a minimal example app for next.js. 

**Summary:**



**Related Issue:**



### Testing
1. Go to `apps/nextjs-example/src/app/page.tsx`
2. Replace the roomUrl constant with a valid room url
3. Run `yarn dev:nextjs-example`
4. Verify that the app renders and connects to the room



### Screenshots/GIFs (if applicable)



### Checklist

        under the terms of the MIT license.

### Additional Information
